### PR TITLE
add animation to nodes and edges

### PR DESCRIPTION
**Pull request by brandonreid**
Animates updates to nodes and edge positions.

### CODE REVIEW

File: src/objects/culling.ts [hunk 1]
```diff
@@ -55,6 +55,11 @@ export async function cull(): Promise<void> {
 }
 
 export async function resumeCulling(): Promise<void> {
+  const application = await waitForApplication()
+
+  if (cullInstance) {
+    cullInstance.cull(application.renderer.screen)
+  }
```
Comment (pleek91) [inline src/objects/culling.ts]: This function needs to be removed. There isn't actually a concept of "pause and resume" There's just culled and unculled. So we should rename the `pauseCulling` function `uncull` and then call `uncull` and `cull`.

File: src/factories/animation.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+  pauseCulling()
+
+  return gsap.to(el, {
+    ...vars,
+    duration,
+    ease: 'power1.out',
+  }).then(() => {
+    resumeCulling()
+  })
```
Comment (pleek91) [inline src/factories/animation.ts]: Couldn't we just await to `gsap.to` and then call cull? 

And just make this return `Promise<void>`. I don't think returning the tween is actually useful.

File: src/factories/edge.ts [hunk 3]
```diff
@@ -10,6 +11,9 @@ import { repeat } from '@/utilities/repeat'
 
 export type EdgeFactory = Awaited<ReturnType<typeof edgeFactory>>
 
+const arrowOffset = 8
+const edgeTargetOffset = 2
+
```
Comment (pleek91) [inline src/factories/edge.ts:16]: Thank you for being a better dev and removing my ugly comments 😂 ❤️

**Review by pleek91 — APPROVED** [review]

File: src/factories/animation.ts (hunk 2 above)
Comment (pleek91) [reply src/factories/animation.ts]: not relevant anymore

File: src/objects/culling.ts (hunk 1 above)
Comment (brandonreid) [reply src/objects/culling.ts]: Went ahead and renamed pauseCulling to uncull and updated. Removed resumeCull. 👍

**Pull request merged**